packages/components/pie-button/src/index.ts

```diff
@@ -1,10 +1,27 @@
-import { LitElement, html } from 'lit';
+import { LitElement, html, css } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
 @customElement('pie-button')
 export class PieButton extends LitElement {
     @property() actionType = 'submit';
 
+    static styles = [css`
+      button {
+        background-color: var(--dt-color-interactive-brand);
```

Comment:
As complexity can vary for different components, the amount of CSS definitions can grow as much. 

Since we're allowed to compose the component styles in the `styles` array, I wonder if it wouldn't be more readable and scalable to define the component CSS in a const outside the class, as the class can contain plenty of code supporting its behavior and lifecycle.

With this approach we benefit from some degree of segregation between style and behavior.

```
const localStyles = css`
    ...
`;

@customElement('pie-button')
export class PieButton extends LitElement {
    @property() actionType = 'submit';

    static styles = [utilityStyles, localStyles]

    render () {
        ...
     }
```

Reply:
Happy to see styling coming along! 🔥

Reply:
Good point. @xander-marjoram and myself discussed this too so we were thinking about importing it from file so we can keep it separated. Similar to what you have but within its own file instead then used with the styles array like you suggested.